test/test.js

```diff
@@ -135,6 +139,7 @@ suite('GaiaHeader', function() {
 
     test('It shouldn\'t run if there is no title `textContent` (ie. before l10n)',
       function(done) {
+      // NOTE: Headings should not be empty.
```

Comment:
Header is not empty but the h1 heading element is, which is not valid from the a11y standpoint.